.github/workflows/get-credentials-gke-it.yml

```diff
@@ -0,0 +1,99 @@
+name: get-credentials-gke Integration
+
+on:
+  push:
+    paths:
+    - 'get-credentials-gke/**'
+    - '.github/workflows/get-credentials-gke**'
+  pull_request:
+    paths:
+    - 'get-credentials-gke/**'
+    - '.github/workflows/get-credentials-gke**'
+
+jobs:
+  gcloud:
+    name: with setup-gcloud & kubectl
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master # Set up ADC for credentials
+      with:
+        service_account_key: ${{ secrets.GKE_AUTH_SA_KEY_JSON }}
+        export_default_credentials: true
+    - id: deploy
+      uses: ./get-credentials-gke
+      env:
+        GCLOUD_PROJECT: ${{ secrets.GKE_AUTH_PROJECT_ID }}
+      with:
+        project_id: ${{ secrets.GKE_AUTH_PROJECT_ID }}
+        name: ${{ secrets.GKE_AUTH_CLUSTER_NAME }}
+        location: ${{ secrets.GKE_AUTH_CLUSTER_LOCATION }}
+        use_auth_provider: true
+    - name: test-kubectl
+      run: |
+        pods=$(kubectl get pods -o=jsonpath='{$.items[*]}')
```

Comment:
`kubectl config view` would just output the current kubectx, it wont actually connect to cluster. For integration I think it would be better if it auths with the cluster. Maybe we can a 2-5 retry incase of ephemeral network issue? The appengine IT uses a similar approach
https://github.com/GoogleCloudPlatform/github-actions/blob/8b66b2e3e755acd3aac3179ea6a38ac4789db8a2/.github/workflows/appengine-deploy-it.yml#L56